Tests/CodeGenerateTests/CodeGenerateTests.swift

```diff
@@ -1,17 +1,97 @@
 import XCTest
-@testable import SmokeFrameworkApplicationGenerate
-
+@testable import SmokeFrameworkApplicationGenerateCommand
 class CodeGenerateTests: XCTestCase {
-    func testExample() {
-        print("This is an example test")
+
+    func testExecutionPlanServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/ExecutionPlanService"
+        let openAPIFilePath = "/Users/mndzup/Desktop/Service_OpenAPI/ExecutionPlanService"
+        let swaggerCommand = ["--base-file-path", swaggerFilePath, "--swagger-version", "2"]
+        let openAPICommand = ["--base-file-path", openAPIFilePath, "--swagger-version", "3"]
+        let fileManager = FileManager()
+
+        SmokeFrameworkApplicationGenerateCommand.main(swaggerCommand)
+        SmokeFrameworkApplicationGenerateCommand.main(openAPICommand)
+        
+        let pass = fileManager.contentsEqual(atPath: swaggerFilePath + "/Sources/", andPath: openAPIFilePath + "/Sources/")
+        XCTAssertTrue(pass)
+    }
+    
+    func testContentIngressServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/ContentIngressService"
+        let openAPIFilePath = "/Users/mndzup/Desktop/Service_OpenAPI/ContentIngressService"
+        let swaggerCommand = ["--base-file-path", swaggerFilePath, "--swagger-version", "2"]
+        let openAPICommand = ["--base-file-path", openAPIFilePath, "--swagger-version", "3"]
+        let fileManager = FileManager()
+
+        SmokeFrameworkApplicationGenerateCommand.main(swaggerCommand)
+        SmokeFrameworkApplicationGenerateCommand.main(openAPICommand)
+        
+        let pass = fileManager.contentsEqual(atPath: swaggerFilePath + "/Sources/", andPath: openAPIFilePath + "/Sources/")
+        XCTAssertTrue(pass)
+    }
+    
+    func testPlaybackAssetsServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/PlaybackAssetsLambda"
+        let openAPIFilePath = "/Users/mndzup/Desktop/Service_OpenAPI/PlaybackAssetsLambda"
+        let swaggerCommand = ["--base-file-path", swaggerFilePath, "--swagger-version", "2"]
+        let openAPICommand = ["--base-file-path", openAPIFilePath, "--swagger-version", "3"]
+        let fileManager = FileManager()
+
+        SmokeFrameworkApplicationGenerateCommand.main(swaggerCommand)
+        SmokeFrameworkApplicationGenerateCommand.main(openAPICommand)
+        
+        let pass = fileManager.contentsEqual(atPath: swaggerFilePath + "/Sources/", andPath: openAPIFilePath + "/Sources/")
+        XCTAssertTrue(pass)
+    }
+    
+    func testPlaybackObjectProcessorServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/PlaybackObjectProcessorService"
```

Comment:
There is a bit of duplicate code in each unit test.  Can we extract the common portions into a shared function?  It looks like only the `swaggerFilePath` and openAPIFilePath` differ for each test.

Reply:
You're absolutely right, I will work on this.